### DISCUSSION TITLE
500 error when not using the default DB_PASSWORD in .env

### DISCUSSION THREAD

**Discussion by ricemark20** [immich-app/immich#4120] [Q&A] [answered]
[SOLVED] I get this error when I use something other than the default DB_PASSWORD of postgres.

` 500 connect ECONNREFUSED 172.24.0.8:3001
`

![image](https://github.com/immich-app/immich/assets/46001899/578a6a6c-233b-48cc-b194-74c47d26f17d)

**Comment by alextran1502** [comment]
You problem having the instance started with a default password, then you try to change it later, is that correct?

If so, you can remove the database volume and restart Immich stack, it will have the new password

**Reply by jrasm91** [reply]
That will also delete your database though, so be careful if you do that.

You can also follow the postgres docs for changing a password. Once a database has been initialized updating the environment variable won't magically change the password.

**Comment by ricemark20** [comment]
> You problem having the instance started with a default password, then you try to change it later, is that correct?
> 
> If so, you can remove the database volume and restart Immich stack, it will have the new password

I am new to Docker, so how do I remove the database volume?

**Reply by bo0tzz** [reply, marked as answer]
https://immich.app/docs/faq#how-can-i-purge-data-from-immich

**Reply by ricemark20** [reply]
I used 
`docker volume prune -f` and `docker volume rm immich-app_pgdata` to remove the Postgress data.

Thank you @bo0tzz for the information.

**Comment by ricemark20** [comment]
> That will also delete your database though, so be careful if you do that.
> 
> That will also delete your database though, so be careful if you do that.  You can also follow the postgres docs for changing a password. Once a database has been initialized updating the environment variable won't magically change the password.

Point taken, but this is a brand new instance that I got around to starting in a VM, so this is not a problem.   It is a good warning.
